lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -279,6 +282,18 @@ CORE_SESSION_TIMEOUT:
     pageId: pyi-timeout-unrecoverable
   parent: END_JOURNEY
 
+EVALUATE_GPG45_SCORES:
+  response:
+    type: process
+    lambda: evaluate-gpg45-scores
+  events:
+    end:
+      targetState:
+        IPV_SUCCESS_PAGE
+    next:
+      targetState:
+        PYI_NO_MATCH
```

Comment:
It doesn't make sense to do it here, but I'm going to add a note onto the next ticket to revise these events to `profile-matched` and `no-profile-matched`!

Reply:
that would be a big improvement!